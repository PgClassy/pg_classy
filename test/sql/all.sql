\set ECHO none
\i test/pgxntool/setup.sql

/*
SET client_min_messages = debug;
SELECT no_plan();
SELECT * FROM _trunklet_test.test_process();
\du
*/
-- Needed for now due to bug in pgtap-core.sql
SET client_min_messages = WARNING;

SET search_path = _classy_test, tap, "$user";
SELECT * FROM runtests( '_classy_test'::name );
