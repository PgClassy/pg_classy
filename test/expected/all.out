\set ECHO none

psql:test/pgxntool/tap_setup.sql:3: NOTICE:  42P06: schema "tap" already exists, skipping
LOCATION:  CreateSchemaCommand, schemacmds.c:115
psql:test/pgxntool/tap_setup.sql:5: NOTICE:  42710: extension "pgtap" already exists, skipping
LOCATION:  CreateExtension, extension.c:1535
    # Subtest: _classy_test.test_instantiate()
    ok 1 - Get test data set "base" for table _classy._class
    ok 2 - SELECT classy.instantiate('pg_classy test create function', 1, '{"body": "SELECT a + b", "options": "", "returns": "int", "language": "sql", "arguments": "a int, b int", "comment_code": "", "function_name": "pg_temp.instantiate_test_function"}')
    1..2
ok 1 - _classy_test.test_instantiate
    # Subtest: _classy_test.test_schemas()
    ok 1 - Role public should be granted USAGE on schema classy
    ok 2 - Role public should be granted no privileges on schema _classy
    1..2
ok 2 - _classy_test.test_schemas
1..2
